.github/workflows/pull-request.yaml

```diff
@@ -140,6 +140,10 @@ jobs:
           ref: ${{ github.event.pull_request.head.sha }}
       - run: rustup toolchain install stable --profile minimal
       - uses: foundry-rs/foundry-toolchain@v1
+        with:
+          # the latest version introduced a bug caused forked node tests to fail
+          # only switch back to latest stable version after it was fixed in anvil
```

Comment:
is there a link we can put here to track this?

Reply:
AFAICS this has not been reported, yet. I wanted to first unblock our test suite before investigating some more and opening an issue. I think with `git bisect` we should be able to find the culprit. 🤞 
The annoying part is that this happens constantly in CI but never locally. So I suspect just reporting the issue without flagging some commit might not result in a quick resolution of the issue.